content/includes/veteran-navigation.html

```diff
@@ -227,22 +236,34 @@
         <li>
           <button
             class="vetnav-level1 vetnav-trigger"
+            role="button"
             aria-controls="vetnav-benefits"
-            aria-haspopup="true">Manage your health &amp; benefits</button>
+            aria-haspopup="true"
+            onClick="window.dataLayer.push({ event: 'nav-header-top-level' });">Manage your health &amp; benefits</button>
           <div class="vetnav-panel" id="vetnav-benefits" hidden>
             <ul role="menu" aria-label="Manage your health and benefits">
-              <li><a class="login-required" href="/health-care/prescriptions">Refill prescriptions</a></li>
-              <li><a class="login-required" href="/health-care/messaging">Message your health care team</a></li>
-              <li><a href="/health-care/schedule-an-appointment/">Schedule a VA appointment</a></li>
-              <li><a class="login-required" href="/health-care/health-records">Get your VA health records</a></li>
-              <li><a class="login-required" href="/track-claims">Check claim and appeal status</a></li>
-              <li><a class="login-required" href="/education/gi-bill/post-9-11/ch-33-benefit">Check Post-9/11 GI Bill benefits</a></li>
+              <li><a class="login-required" href="/health-care/prescriptions" onClick="reportHeaderNav('Manage Health and Benefits');">Refill prescriptions</a></li>
+              <li><a class="login-required" href="/health-care/messaging" onClick="reportHeaderNav('Manage Health and Benefits');">Message your health care team</a></li>
+              <li><a href="/health-care/schedule-an-appointment/" onClick="reportHeaderNav('Manage Health and Benefits');">Schedule a VA appointment</a></li>
+              <li><a class="login-required" href="/health-care/health-records" onClick="reportHeaderNav('Manage Health and Benefits');">Get your VA health records</a></li>
+              <li><a class="login-required" href="/track-claims" onClick="reportHeaderNav('Manage Health and Benefits');">Check claim and appeal status</a></li>
+              <li><a class="login-required" href="/education/gi-bill/post-9-11/ch-33-benefit" onClick="reportHeaderNav('Manage Health and Benefits');">Check Post-9/11 GI Bill benefits</a></li>
             </ul>
           </div>
         </li>
 
-        <li><a href="/facilities" class="vetnav-level1">Find VA locations</a></li>
+
+        <li><a href="/facilities/" class="vetnav-level1" onClick="reportHeaderNav('Find VA locations');">Find VA locations</a></li>
       </ul>
     </nav>
   </div>
 </div>
+
+<script>
+function reportHeaderNav(navpath) {
+  window.dataLayer.push({
+    event: 'nav-header-link',
+    'nav-path': navpath
+  });
+}
```

Comment:
It may be worth considering at some point using JS to attach event listeners:

``` html
<li><a class="login-required" href="/health-care/prescriptions" data-navpath="Manage Health and Benefits">Refill prescriptions</a></li>
```

``` javascript
document.querySelectorAll('#vetnav li > a').forEach(function(element){
    element.addEventListener('click', function(e){
        const navpath = element.dataset.navpath;
        window.dataLayer.push({
         event: 'nav-header-link',
         'nav-path': navpath
       });
    });
});
```

Reply:
yeah, i had that originally but the but i felt like the `onClick` versus the `data-navpath` didn't save much in the readability of the HTML and was a little more clear to show something happened on click right in the element. 

What i _really_ wanted to do was move all the nav elements into some YAML and then use liquid templates to make all the HTML so we could separate the data from the presentation more cleanly. But, it just sort of grew into a furball. But i think the header nav could probably use a good refactor. I saw this first PR as an incremental step towards that.

Reply:
Yeah I know you mean, I agree. This looks good the way it is. The navigation has gone through all sorts of changes lately with accessibility and viewports and it would be nice to see it all simplified at some point.